src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -327,14 +355,10 @@ public Bridge selectLeastLoadedNearby(List<Bridge> bridges,
         {
             return null;
         }
-
-        return a1(bridges, conferenceBridges, participantRegion).orElse(
-            a2(bridges, conferenceBridges, participantRegion).orElse(
-                b1(bridges, conferenceBridges, participantRegion).orElse(
-                    c1(bridges, conferenceBridges, participantRegion).orElse(
-                        c2(bridges, conferenceBridges, participantRegion).orElse(
-                            d1(bridges, conferenceBridges, participantRegion).orElse(
-                                d2(bridges, conferenceBridges, participantRegion).orElse(null)))))));
+        else
+        {
+            return a1(bridges, conferenceBridges, participantRegion);
```

Comment:
Using `orElseGet` instead of `orElse` will give you lazy evaluation

Reply:
I reverted the commit that refactored the code and replaced the orElse with orElseGet.